{{cookiecutter.project_slug}}/bitbucket-pipelines.yml

```
@@ -0,0 +1,61 @@
+image: python:3.10
+
+definitions:
+  services:
+    docker:
+      memory: 3072
+    postgres:
+      image: postgres:14
```

Let's keep this generic, it can always be fixed on the project:
```suggestion
      image: postgres:latest
```